llvm/lib/Target/SBF/SBFInstrInfo.td

```diff
@@ -263,7 +265,7 @@ class ALU_RR<SBFOpClass Class, SBFArithOp Opc,
   let SBFClass = Class;
 }
 
-multiclass ALU<SBFArithOp Opc, string Mnemonic, SDNode OpNode> {
+multiclass ALU<SBFArithOp Opc, string Mnemonic, SDNode OpNode, bit UseImmPat = 1> {
```

Comment:
LLVM Coding standard nit: (80-column rules) Please move the new parameter to the next line, aligned with the `<`'.